site/src/style/variables.less

```diff
@@ -18,3 +18,12 @@
 @jumbotron-color:       white;
 
 @section-vertical-padding: 30px;
+
```

Comment:
Can't argue with the science!

I tried boosting the base font size, but funnily enough that just made everything bigger ;-) I'm happy to go with the standard.